sourcecode/hub/resources/css/theme.scss

```diff
@@ -19,6 +19,10 @@
         background-color: $green-200;
     }
 
+    .no-content-found-title {
+        color: $black;
```

Comment:
This should use the default text colour in the current theme, or one of the secondary colours, which may not necessarily be black.